templates/distribution/_helpers.tpl

```diff
@@ -53,23 +65,25 @@
 {{- end }}
 {{- end -}}
 
-{{ define "pomeriumHost" }}
-  {{- template "ingressHost" (dict "module" "auth" "package" "pomerium" "prefix" "pomerium.internal." "spec" .spec) -}}
-{{ end }}
-
-{{ define "ingressAuthUrl" -}}
-"https://{{ template "pomeriumHost" . }}/verify?uri=$scheme://$host$request_uri"
+{{/* ingressTlsAuth { module: <module>, package: <package>, prefix: <prefix>, spec: "." } */}}
+{{- define "ingressTlsAuth" -}}
+{{ if eq .spec.distribution.modules.ingress.nginx.tls.provider "none" -}}
+  {{ else }}
```

Comment:
if you don't want to handle the case when provider is none, maybe is better this way:

```suggestion
{{ if ne .spec.distribution.modules.ingress.nginx.tls.provider "none" -}}
```

Reply:
I copy pasted the function from ingressTls, we should change the logic in that function also. Maybe in a next iteration, I already tested some scenarios